apps/meteor/server/services/upload/service.ts

```diff
@@ -0,0 +1,39 @@
+import type { IMessage, IUpload } from '@rocket.chat/core-typings';
+import { Meteor } from 'meteor/meteor';
+
+import { ServiceClassInternal } from '../../sdk/types/ServiceClass';
+import type {
+	ISendFileLivechatMessageParams,
+	ISendFileMessageParams,
+	IUploadFileParams,
+	IUploadService,
+} from '../../sdk/types/IUploadService';
+import { FileUpload } from '../../../app/file-upload/server';
+
+export class UploadService extends ServiceClassInternal implements IUploadService {
+	protected name = 'upload';
+
+	async uploadFile({ buffer, details }: IUploadFileParams): Promise<IUpload> {
+		const fileStore = FileUpload.getStore('Uploads');
```

Comment:
If this is an "uploads" service, we may have this as a property instead of a string. So we call `this.store`

Reply:
once we move everything to this service, yes, agreed. right now the "stores" are created async, which then requires us from using `getStore` every time.. it is not something expensive to call `getStore` but I agree it makes sense to have it as a local variable inside the service

Reply:
To not remove your approval, I'll get this done for the future 👀